app/src/main/java/edu/uc/hickmadc/customnotifications/dto/Group.kt

```diff
@@ -1,7 +1,13 @@
 package edu.uc.hickmadc.customnotifications.dto
 
-data class Group (var name : String, var desc : String, var active : Boolean, var same_schedule : Boolean, var group_id:Int=0) {
+data class Group(
+    var groupName: String,
+    var groupDescription: String,
+    var groupActive: Boolean,
+    var sameSchedule: Boolean,
+    var groupId: Int = 0
+) {
     override fun toString(): String {
-        return name
+        return groupName
```

Comment:
Ditto comments above.  "name" has the context of the class "Game", so no need to repeat that in the variable name.